### PR TITLE
Bootcamp/jan2020

### PR DESCRIPTION
**Pull request by sheenarbw**
I'm just looking for feedback for now. This still needs some work. Please look at:

content/projects/katas/level-0/_index.md
content/projects/katas/level-1/_index.md
content/projects/katas/level-2/_index.md

Basically the projects we use for checking coding aptitude during bootcamp for web devs, data eng and java applicants are the same:
we give them 3 levels of katas:
- level 0: this is seriously basic. They submit a link to a repo 
- level 1: this kata gets them to practice git feature branching as well as a few harder code problems
- level 2: not compulsory for bootcamp, but included to see if anyone gets there. Like level 1 only harder

Why:
we used to just have 2 kata projects in boot, but people struggled to finish everything. We think we can make good decisions with less code. So some problems were moved from level 1 to level 2 to make the bootcamp a bit more reasonable, but there is still space for people to stretch a bit

## I solemnly swear that:

- [ ] I ran the hugo server and looked at my changed in the browser with my own eyes
- [ ] I ran the linter and there were no errors
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code

### CODE REVIEW

File: content/projects/bootcamp-writing-assignment/_index.md [hunk 1]
```diff
@@ -29,56 +31,57 @@ If you are using ACode on your android device, you can press the "play" button i
...
-2. A lot of people dont really sit down and think about what they are trying to do in the long run. It's a really useful excercise
 
 ### The questions
 
-Please create a separate file for each of these. Remember to pay attention to your sentence structure, spelling and grammar.
+Please create a separate file for each of these. Remember to pay attention to your sentence structure, spelling and grammar. 
+
+If you need a little help with your grammar, [Grammarly](https://app.grammarly.com/) is a really useful service. Being able to communicate clearly and professionally in text is a super power.
 
 #### 1. Your history [200 - 500 words]
```
Comment (ry-oc) [inline content/projects/bootcamp-writing-assignment/_index.md:41]: Can we add something to this about time/financial commitments?

Reason being we need to know if they are freelancing/working/in uni full time or anything. Also we need to know if they can afford to work for a small stipend.

**Review by ry-oc — COMMENTED** [review]
Added a bunch of comments, if anything is not relevant or a bad ide feel free to ingnore it :)

File: content/topics/messaging/umuzi-recruit-user-journey/_index.md [hunk 2]
```diff
@@ -55,87 +47,80 @@ PLEASE NOTE: the bootcamp should be treated like a full time job! Spend at least
...
+### 6. External work experience
 
-Once you are done with your capstone project you will be allowed to join a product team.
-
-This will polish your skills, working on a real team towards real goals is something else. It's great preparation for the next phase.
-
-### 8. External work experience
-
-After the first 9 months of your contract with us, we start watching you extra closely and preparing you for job interviews.
+After the first 9 months of your contract with us, we start watching you extra closely and preparing you for job interviews. This
```
[the remark this answers is not in the document]
Comment (sheenarbw) [reply content/topics/messaging/umuzi-recruit-user-journey/_index.md]: You wasted your time. Read the PR description.

File: content/topics/messaging/umuzi-recruit-user-journey/_index.md [hunk 3]
```diff
@@ -55,87 +47,80 @@ PLEASE NOTE: the bootcamp should be treated like a full time job! Spend at least
...
+Here at Umuzi, we require professional behavior. 
+
+- A professional shows up and does the work
+- A professional pursues understanding
+- A professional tries to solve their own problems
+- A professional asks for help when they need it
+- A professional helps the people around them. That's right, if someone asks for help and you help them then that's a win
 
-### 4. Contract signing
+### 3. Contract signing
```
[the remark this answers is not in the document]
Comment (sheenarbw) [reply content/topics/messaging/umuzi-recruit-user-journey/_index.md:62]: this wasn't what I asked for a review on...

File: content/projects/bootcamp-writing-assignment/_index.md (hunk 1 above)
Comment (sheenarbw) [reply content/projects/bootcamp-writing-assignment/_index.md:41]: I thought you said you would add the extra questions?

**Pull request merged**